src/libs/ReportUtils.ts

```diff
@@ -4242,8 +4284,10 @@ function updateOptimisticParentReportAction(parentReportAction: OnyxEntry<Report
  * @param text - Text of the comment
  * @param parentReportID - Report ID of the parent report
  * @param createdOffset - The offset for task's created time that created via a loop
+ * @param delegate - Email of the delegate
  */
 function buildOptimisticTaskCommentReportAction(
+    delegate: string,
```

Comment:
Should we remove delegate params and get delegateID inside this function?

Reply:
@allgandalf Yeah. But the value of this params isn't change. I don't see any reason why we should pass it as a param

Reply:
I still see we are using Onyx.connect in reportUtils. On the other hand, we can create a new function to get delegateEmail and use in reportUtils